multimedia/minisatip/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2010-2012 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minisatip
+PKG_VERSION:=0.5
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/catalinii/minisatip/archive
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
```

Comment:
This is a no go, the tarball filename will potentially clash with other tarballs using this naming and there's no way to tell without extracting what it is. Use this method instead:
https://github.com/openwrt/packages/blob/master/sound/fdk-aac/Makefile#L15